docs/developer/getting-started/debugging.asciidoc

```diff
@@ -27,11 +37,8 @@ APM config option. To activate the APM agent, use the
 https://www.elastic.co/guide/en/apm/agent/nodejs/current/configuration.html#active[`active`]
 APM config option.
 
-All config options can be set either via environment variables, or by
-creating an appropriate config file under `config/apm.dev.js`. For
-more information about configuring the APM agent, please refer to
-https://www.elastic.co/guide/en/apm/agent/nodejs/current/configuring-the-agent.html[the
-documentation].
+All config options can be set by
+creating an appropriate config file under `config/apm.dev.js`. 
```

Comment:
Should we document that it is also possible to configure the agent via `kibana.yml`?

Reply:
@pgayvallet we can add that later, once we're ok with enabling in prod.